collection-scripts/gather_db

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# When called from the shell directly
+if [[ -z "$DIR_NAME" ]]; then
+    CALLED=1
+    DIR_NAME=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+    source "${DIR_NAME}/common.sh"
+fi
+
+# Default DB DUMP Option
+DB_OPT="--single-transaction --complete-insert --skip-lock-tables --lock-tables=0"
+DB_DUMP=${BASE_COLLECTION_PATH}/dbs
+
+function dump_db {
+    local dbpod="$1"
+    local ns="$2"
+    local dbpass="$3"
+    local dbname="$4"
+    [[ -z $dbname ]] && dump="openstack_databases" || dump=$dbname
+    run_bg /usr/bin/oc -n $ns rsh -c galera $dbpod mysqldump -uroot -p$dbpass $DB_OPT $dbname > "$DB_DUMP"/$dump.sql
+}
+
+# If unset use all default services
+if [ "${OPENSTACK_DATABASES-unset}" = "unset" ] || [[ -z "${OPENSTACK_DATABASES[*]}" ]]; then
```

Comment:
?: Isn't `OPENSTACK_DATABASES` a string?  So shouldn't we just use `[[ -z "${OPENSTACK_DATABASES}" ]]`

Reply:
yeah, it's not an array, but only a comma separated list of items, or an empty string, but yes, just a string, so the `[*]` is not required. Fixing in the next iteration.

Reply:
Done